website/app/styles/pages/application/banner.scss

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// BANNER
+
+@use "../../breakpoints" as breakpoint;
+
+.doc-page-banner {
+  position: relative;
+  z-index: 2;
+  display: flex;
+  grid-area: banner;
+  gap: 6px;
+  align-items: center;
+  justify-content: center;
+  padding: 16px 12px;
+  color: #fff;
+  background: linear-gradient(135deg, #4f53d0 0%, #7545cb 100%);
+
+  body:not(.index) & {
+    @include breakpoint.with-fixed-sidebar() {
+      padding-left: var(--doc-page-sidebar-width); // we need to compensate for the fixed position of the sidebar
+    }
+  }
+}
+
+.doc-page-banner__icon {
+  flex: none;
+}
+
+.doc-page-banner__text {
+  margin: 0;
+
+  a {
+    color: inherit;
+
+    &::after {
+      position: absolute;
+      inset: 0;
+      content: "";
```

Comment:
This appears to make the whole banner interactive. I'd like for only the link to be interactive so that this works as users expect it to and to avoid accidental clicks.